src/form/Form.tsx

```diff
@@ -1053,7 +1059,31 @@ function Form({
     }
   };
 
-  function handleActions(setLoader: any) {
+  function setCardElement(key: string, cardElement: any) {
+    setCardElementMappings((cardElementMappings) => ({
+      ...cardElementMappings,
+      [key]: cardElement
+    }));
+  }
+
+  async function handleActions(setLoader: any, formattedFields: any) {
+    // Perform any Stripe payment methods setup here
+    try {
+      await setupPaymentMethods(
```

Comment:
you can't try/catch errors from async functions like this, you'd have to do 

const handleActions = (...) => setupPaymentMethods(...).catch((p) => p);

Reply:
I might be missing a bit of nuance about what exactly this code is trying to do, but you can use try/catch blocks within an async fn - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#promise_rejection

Reply:
I simplified here to remove the throw and catch.  A simple error return was sufficient.